src/schema/v2/artwork/index.ts

```diff
@@ -1446,6 +1446,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      visibilityLevel: {
+        description: "The visibility level of the artwork",
+        type: GraphQLString,
+        resolve: ({ visibility_level }) => visibility_level,
+      },
```

Comment:
Might be nice to add a spec to capture the new field

Something simple like:
https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/__tests__/artwork.test.js#L692-L714 for example!

Reply:
cool cool will do!